spec/features/saml/loa3_sso_spec.rb

```diff
@@ -126,6 +168,40 @@ def sign_out_user
       )
     end
 
+    let(:return_to_verify_button) { t('idv.buttons.return_to_verify') }
+
+    context 'having selected phone option' do
+      let(:phone_confirmed) { false }
+
+      it 'there is a button that will return the user to the activation option page' do
```

Comment:
What do you think about replacing `there is` with `includes`? I like to make tests readable starting from the `it`.